File: SofiForce.Models/Models/ListModels/ClientDocumentListModel.cs
using System;

namespace Models
{
    public class ClientDocumentListModel
    {
        public int ClientDocumentId { get; set; }
        public int? ClientId { get; set; }
        public int? DocumentTypeId { get; set; }
        public string DocumentPath { get; set; }
        public DateTime? UploadDate { get; set; }
        public string DocumentExt { get; set; }
        public int? DocumentSize { get; set; }
        public string DocumentTypeName { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/clientDocumentListModel.ts
export interface ClientDocumentListModel {
    clientDocumentId: number;
    clientId: number | null;
    documentTypeId: number | null;
    documentPath: string;
    uploadDate: Date | null;
    documentExt: string;
    documentSize: number | null;
    documentTypeName: string;
}